fstat/controller.py

```diff
@@ -26,7 +27,7 @@ def index():
 
 @app.route('/login', methods=['POST'])
 def login():
-    return github.authorize(scope="user:email,read:org")
+    return github.authorize(scope="user,user:email,read:org")
```

Comment:
we are storing the profile picture url and name that's why I am using user scope